modules/apache-airflow/README.md

```diff
@@ -65,4 +65,20 @@ YAML
 ## Accessing the web UI
 An `admin` user is created for airflow via the `airflow-admin-user-secret` secret that
 is added to the namespace. Decode the base64 encoded password/username and use it for
-the UI.
\ No newline at end of file
+the UI.
+
+## Building a new image for airflow
+The deployment of our airflow instance depends on a custom apache airflow image being 
+created and pushed to a public available GCHR url. The image is created from the
+`orca-recipes` git repo: <https://github.com/Sage-Bionetworks-Workflows/orca-recipes/tree/main>
+
+1. Update the dockerfile within the orca-recipes repo
+2. Build the new image `docker build .`
+3. Tag the build image with the tag you want to use `docker tag sha256:... ghcr.io/sage-bionetworks-workflows/orca-recipes:0.0.1`
```

Comment:
Ideally in the long term - a github action is set up to do this

Reply:
Agreed! It's not much work to do this manually, but a GH action in the `orca-recipes` repo will make the experience better. I created this ticket to add this logic: https://sagebionetworks.jira.com/browse/IBCDPE-1134